### ISSUE TITLE
Integrate `join` sub-command to join multiple GFF files

### ISSUE THREAD

**Issue by lpantano** [miRTop/mirtop#31]
The main function that does this is already coded here: 

https://github.com/miRTop/mirtop/blob/dev/mirtop/gff/merge.py#L3

`dts` is a dict where keys are `sample name` that could be the basename of the GFF file that are the input to the command, and values is nested dicts with the following structure:

`dicts['colum0']['column4'] = [list1, list2, listn]`

Each `listn` contains 4 elements: `[IDU,NA,EXPRESSION, GFF_LINE]`

Each value represent a variable inside attribute column as explained here:https://github.com/miRTop/incubator/blob/master/format/definition.md , except for the last one that is the entire GFF line.

Noted NA could be an empty value, that is not needed for this specific functionality.

Basically, it's needed a function to read each GFF to have the structure above explained, and the input that to the function `merge`. 

Header needsto me modify to add the:

* CMD info that has the command line used
* COLDATA info needs to be outdated with all the sample names

If this is unclear, just think a way to join two GFF, for instance from: https://github.com/miRTop/mirtop/blob/dev/data/examples/gff/correct_file.gff 

Final function need to be tested and added as a unit test in https://github.com/miRTop/mirtop/blob/dev/test/test_functions.py

Thanks

**Comment by lpantano** [comment]
this was implemented in the in `gff` subcommand.
